src/widgets/sceneviewwidget.cpp

```diff
@@ -756,14 +756,21 @@ void SceneViewWidget::renderCameraUi(iris::SpriteBatchPtr batch)
 	else 
 		text = "orthogonal";
 
-	if (dist(yaw,0) && dist(pitch, -90))
+	if (dist(yaw, 0) && dist(pitch, -90))
 		text += " (top)";
 	if (dist(yaw, 90) && dist(pitch, 0))
-		text += " (side)";
+		text += " (left)";
 	if (dist(yaw, 0) && dist(pitch, 0))
 		text += " (front)";
+
+	if (dist(yaw, 0) && dist(pitch, 90))
+		text += " (bottom)";
+	if (dist(yaw, -90) && dist(pitch, 0))
+		text += " (right)";
+	if (dist(yaw, -180) && dist(pitch, 0))
+		text += " (back)";
```

Comment:
Prefer something like `QString("%1 - %2").arg(orientation).arg(view)` instead of the +=s. Also make those &&s into variables so its `if (LeftOrientation)` or something instead of `if (dist(yaw, 90) && dist(pitch, 0))`